resuscitation.csl

```diff
@@ -0,0 +1,332 @@
+<style class="in-text" version="1.0" page-range-format="minimal-two" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Resuscitation</title>
+    <id>http://www.zotero.org/styles/resuscitation</id>
+    <link href="http://www.zotero.org/styles/resuscitation" rel="self"/>
+    <link href="http://www.zotero.org/styles/current-gene-therapy" rel="template"/>
+    <link href="http://www.resuscitationjournal.com/content/authorinfo" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0300-9572</issn>
+    <eissn>1873-1570</eissn>
+    <summary>Resuscitation is the official monthly interdisciplinary medical journal of the European Resuscitation Council.</summary>
+    <updated>2017-07-17T11:22:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="accessed">cited</term>
+    </terms>
+  </locale>
+  <macro name="page">
+    <choose>
+      <if is-numeric="locator">
+        <label variable="locator" form="short" suffix=" "/>
+        <text variable="locator"/>
+      </if>
+      <else>
+        <text variable="locator"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <choose>
+      <if type="legislation"/>
+      <else-if type="entry-encyclopedia" variable="author" match="all">
+        <names variable="author">
+          <name name-as-sort-order="all" sort-separator=" " initialize-with=""/>
+        </names>
+      </else-if>
+      <else-if variable="author editor composer" match="any">
+        <names variable="author">
+          <name name-as-sort-order="all" sort-separator=" " initialize-with=""/>
+          <label prefix=", "/>
+          <substitute>
+            <names variable="editor"/>
+            <names variable="composer"/>
+          </substitute>
+        </names>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="entry-dictionary entry-encyclopedia">
+        <choose>
+          <if variable="author">
+            <text variable="title"/>
+          </if>
+        </choose>
+      </if>
+      <else-if type="book" match="any">
+        <text variable="title"/>
+      </else-if>
+      <else-if type="post-weblog" match="none">
+        <text variable="title"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="editor-container">
+    <group delimiter=": ">
+      <text term="in" text-case="capitalize-first"/>
+      <group delimiter=". ">
+        <names variable="editor">
+          <name name-as-sort-order="all" sort-separator=" " initialize-with=""/>
+          <label form="short" text-case="capitalize-first" prefix=", "/>
```

Comment:
in the styleguide I'm seeing ", editors", i.e. without form="short" and "capitalize-first". Did you overlook this or do articles diverge?